docs/literate/src/files/upwind_fdsbp.jl

```diff
@@ -0,0 +1,61 @@
+#src # Upwind FD SBP schemes
+
+# General tensor product SBP methods are supported via the `DGMulti` solver
+# in a reasonably complete way, see [the previous tutorial](@ref DGMulti_2).
+# Nevertheless, there is also experimental support for SBP methods with
+# other solver and mesh types.
+
+# The first step is to set up an SBP operator. A classical (central) SBP
+# operator can be created as follows.
+using Trixi
+D_SBP = derivative_operator(SummationByPartsOperators.MattssonNordström2004(),
+                            derivative_order=1, accuracy_order=2,
+                            xmin=0.0, xmax=1.0, N=11)
+# Instead of prefixing the source of coefficients `MattssonNordström2004()`,
+# you can also load the package SummationByPartsOperators.jl. Either way,
+# this yields an object representing the operator efficiently. If you want to
+# compare it to coefficients presented in the literature, you can convert it
+# to a matrix.
+Matrix(D_SBP)
+
+# Upwind SBP operators are a concept introduced in 2017 by Ken Mattsson. You can
+# create them as follows.
+D_upw = upwind_operators(SummationByPartsOperators.Mattsson2017,
+                         derivative_order=1, accuracy_order=2,
+                         xmin=0.0, xmax=1.0, N=11)
+# Upwind operators are derivative operators biased towards one direction.
+# The "minus" variants has a bias towards the left side.
```

Comment:
Maybe you could clarify once what is meant exactly by "biased towards one direction, e.g., something along the lines "...has a bias towards the left side, i.e., it will consider more information from the left side instead of the right side to determine the derivative at a given position". Just to be more in the spirit of a "tutorial" :-)

Reply:
Also, from the printout out seems like the left side (coeff = 5.0) is actually *less* favored than the right side (coeff = 15.0), e.g., in line 3:
```
5.0  -20.0   15.0    0.0    0.0    0.0    0.0    0.0    0.0    0.0   0.0
```
Am I just getting this wrong or is left and right flipped? Maybe a word on this would help the naive reader like myself...

Reply:
Good point. I added something in [664538c](https://github.com/trixi-framework/Trixi.jl/pull/1283/commits/664538c45335ce5e4686c8d8c16fbd1168f29509). Does that help?

Reply:
It does, thank you! Apparently I miscounted where the diagonal is 🙈.